sdk/internal/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.4.1 (Unreleased)
+## 1.5.1 (Unreleased)
```

Comment:
The `1.5.1` placeholder can be removed for now. It will automatically be added post release of `1.5.0`.